06-TimeManagement/utils/program_06.c

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ *  program_06.c - Test time
+ *
+ *  Copyright (C) 2019 Andrey Pahomov <pahomov.and@gmail.com>
+ */
+#include <time.h>
+#include <stdio.h>
+#include <asm-generic/param.h>
+
+int main(int argc, char *argv[])
+{
+	struct timespec tv;
+	time_t sec;
+
+	clock_gettime(CLOCK_REALTIME, &tv);
+
+	time(&sec);
+	printf("Date: %s", ctime(&sec));
+
+	printf("%10ld sec, %9ld ns\n",
```

Comment:
Value of seconds and nanoseconds can be for another date in such implementation.